packages/notebook-extension/schema/tracker.json

```diff
@@ -833,7 +833,7 @@
     "windowingMode": {
       "title": "Windowing mode",
       "description": "'defer': Improve loading time - Wait for idle CPU cycles to attach out of viewport cells - 'full': Best performance with side effects - Attach to the DOM only cells in viewport - 'none': Worst performance without side effects - Attach all cells to the viewport",
-      "enum": ["defer", "full", "none"],
+      "enum": ["defer", "full", "none", "contentVisibility"],
       "default": "full"
```

Comment:
Is there documentation or previous discussion to better understand why the default is set to 'full'?

Are there plans to default to 'contentVisibility'?

Reply:
We started seeing issues with 'full' but at 4.4.x. Up until 4.3.8 we do not see problems. Please see : https://github.com/jupyterlab/jupyterlab/issues/17815 for more information